src/components/Results.tsx

```diff
@@ -0,0 +1,108 @@
+import { useSurveyLanguages } from '../hooks';
+import { SectionHeader } from './SectionHeader';
+import { ResponseOption } from '../types';
+import { BoldText } from './Text';
+import { Wrapper, WrapperHorizontal, WrapperRow } from './Wrapper';
+
+type Props = { responseOptions: ResponseOption[]; selectedOption: string };
+type SingleProps = {
+  option: ResponseOption & { index: number };
+  selected: boolean;
+  totalCount: number;
+};
+
+const getPercent = (partial: number, total: number) => {
+  if (!total) {
+    return '0%';
+  }
+  return `${((100 * partial) / total).toFixed(0)}%`;
+};
+
+const getCountLabel = (partial: number, total: number) => {
+  return `${partial} / ${getPercent(partial, total)}`;
+};
+
+const SingleResult = ({ option, selected, totalCount }: SingleProps) => {
+  const percentString = getPercent(option.votesCount, totalCount);
+  const width = percentString === '0%' ? barBorderRadius * 2 : percentString;
+  return (
+    <WrapperRow spaceBetween style={styles.container}>
+      <View style={styles.labelContainer}>
+        <BoldText small>{getAnswerLabel('de', option.index)}</BoldText>
+        <BoldText small>{getAnswerLabel('pl', option.index)}</BoldText>
+      </View>
+      <WrapperHorizontal style={styles.barContainer}>
+        <View
+          style={[
+            styles.baseBarStyle,
+            // eslint-disable-next-line react-native/no-inline-styles
+            {
+              opacity: selected ? 1 : 0.5,
+              width
+            }
+          ]}
+        />
+      </WrapperHorizontal>
+      <View style={styles.countContainer}>
+        <BoldText>{getCountLabel(option.votesCount, totalCount)}</BoldText>
+      </View>
+    </WrapperRow>
+  );
+};
+
+export const Results = ({ responseOptions, selectedOption }: Props) => {
+  const languages = useSurveyLanguages();
+  const sortedOptions = responseOptions
+    .map((option, index) => ({ ...option, index }))
+    .sort((a, b) => -a.votesCount + b.votesCount);
+
+  const totalCount = responseOptions.reduce(
+    (prev, currentOption) => prev + currentOption.votesCount,
+    0
+  );
+
+  const title = combineLanguages(languages, texts.survey.result);
+
+  return (
+    <>
+      {!!title && <SectionHeader title={title} />}
+      <Wrapper>
+        {sortedOptions.map((option) => (
+          <SingleResult
+            key={option.id}
+            option={option}
+            selected={selectedOption === option.id}
+            totalCount={totalCount}
+          />
+        ))}
+      </Wrapper>
+    </>
+  );
+};
+
+const barBorderRadius = 4;
+
+const styles = StyleSheet.create({
+  barContainer: { flex: 1, justifyContent: 'center' },
+  baseBarStyle: {
+    backgroundColor: colors.primary,
+    borderRadius: 4,
+    height: normalize(26)
+  },
+  container: {
+    marginBottom: normalize(12)
+  },
+  countContainer: {
+    position: 'absolute',
+    right: 0,
+    top: normalize(7)
+  },
+  labelContainer: {
+    width: 100
```

Comment:
this fix 100 fits for polish now, right?

Reply:
added a comment.

the 100 makes the width slightly bigger than the texts (both german and polish), so that there are no forced linebreaks and the bars have no offsets between each other